cms/envs/common.py

```diff
@@ -173,8 +173,7 @@
 FEATURES = {
     'GITHUB_PUSH': False,
 
-    # for consistency in user-experience, keep the value of the following 3 settings
-    # in sync with the ones in lms/envs/common.py
+    # See annotations in lms/envs/common.py for details.
     'ENABLE_DISCUSSION_SERVICE': True,
     'ENABLE_TEXTBOOK': True,
```

Comment:
Great.  Thank you.  Can you just add this one additional comment and I can approve/merge.  Thanks again.
```suggestion
    'ENABLE_DISCUSSION_SERVICE': True,
    # See annotations in lms/envs/common.py for details.
    'ENABLE_TEXTBOOK': True,
```